tests/flow/test_procedures.py

```
@@ -474,3 +489,34 @@ def test12_list_functions(self):
+            "properties",
+            "types",
+            "options",
+            "language",
+            "stopwords",
+            "entitytype",
+            "status",
+            "info",
+        ]
+        self.env.assertEqual(result[0][: len(expected_header)], expected_header)
+
+        row_by_column = dict(zip(result[0], result[1][0]))
```

Ruff (0.15.14)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)